carma_messenger_vehicle_plugin/src/resources/msger_veh_cfg.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Would be good to document the expected format of this JSON file somewhere, preferably in a doc comment close to where it's loaded.

Reply:
I've created a [confluence page](https://usdot-carma.atlassian.net/wiki/spaces/CRMSIM/pages/3153788930/Setup+Instruction+Documentation+CARMA+Messenger+Vehicle+Plugin) to describe the elements in this JSON file